packages/devtools_server/lib/src/server.dart

```diff
@@ -130,10 +130,7 @@ final argParser = ArgParser()
 /// Wraps [serveDevTools] `arguments` parsed, as from the command line.
 ///
 /// For more information on `handler`, see [serveDevTools].
-Future<HttpServer> serveDevToolsWithArgs(
-  List<String> arguments, {
-  shelf.Handler handler,
```

Comment:
Is `handler` no longer required here? I thought it was added to support other tools (webdev or something interal?) that consume the server package. If so, this may be a breaking change to them.

Reply:
Ah, I didn't see any uses in this repo, but didn't realize that webdev was relying on it. I'll revert this part of the change.

Reply:
I don't know for sure, it just seemed to ring a bell. Was added in https://github.com/flutter/devtools/pull/1039 but there's not much info so I'm not sure why I thought this (maybe from a discussion on a VC?).

If it is needed, it might be worth having a comment to avoid it being cleaned up.

Reply:
(reverted this part of the change)